proto/hub-genesis/params.proto

```diff
@@ -8,18 +8,6 @@ import "gogoproto/gogo.proto";
 // Params holds parameters for the hub-genesis module.
 message Params {
   option (gogoproto.goproto_stringer) = false;
```

Comment:
could we not just remove `Params` altogether?

Reply:
Yes you're right it should be removed
but is it ok if I do it in a followup PR? This PR is currently all tested etc
https://github.com/dymensionxyz/dymension-rdk/issues/452
I have made an issue for myself
Thanks